docker-compose.yml

```diff
@@ -37,22 +35,21 @@ services:
   database:
     image: postgres:latest
     container_name: database
-    restart: on-failure
+    restart: always
```

Comment:
Why always and not on-failure?

Reply:
no particular reason, it doesn't really make a difference in our project's context I guess. I was just trying out stuff